### PR TITLE
Extend PSI easyblock to use PCMSolver and CheMPS2

### PR DESCRIPTION
**Pull request by wpoely86**
@boegel please review

### CODE REVIEW

File: easybuild/easyblocks/p/psi.py [hunk 1]
```diff
@@ -147,6 +147,15 @@ def configure_step(self):
             if get_software_root('impi'):
                 self.cfg.update('configopts', " -DENABLE_CSR=ON -DBLAS_TYPE=MKL")
 
+            if self.cfg.name == "PSI4":
```
Comment (boegel) [inline easybuild/easyblocks/p/psi.py]: we also use this in `PSI-4.0b6-20160201-intel-2016a-mt-Python-2.7.11.eb`?

**Comment by boegel** [comment]
@wpoely86 can you make a companion easyconfigs PR that cleans up the easyconfigs?

File: easybuild/easyblocks/p/psi.py (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyblocks/p/psi.py]: yes, but I'm not gonna touch the old PSI anymore. And it's version hell anyway if I do that.

**Comment by wpoely86** [comment]
Once the original PR is merged yes.

**Comment by boegel** [comment]
tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/3312, lgtm, thanks @wpoely86!

**Pull request merged**